test/WebSites/ControllersFromServicesWebSite/project.json

```diff
@@ -1,7 +1,6 @@
 {
   "buildOptions": {
-    "emitEntryPoint": true,
-    "preserveCompilationContext": true
```

Comment:
The compilation context \ `DependencyContext` for this app isn't used. We use the one from the ClassLibrary. This enforces the expectation.